src/Microsoft.DotNet.Cli.Utils/PathUtility.cs

```diff
@@ -257,5 +257,16 @@ public static string GetFullPath(string path)
 
             return Path.GetFullPath(path);
         }
+
+        public static string GetCurrentDirectoryWithDirSeparator()
```

Comment:
I prefer the "WithDirSeparator" because it's clear without looking at the implementation what the method is doing.